### DISCUSSION TITLE
Flet needs a CODESTYLE!!!

### DISCUSSION THREAD

**Discussion by mrkito** [flet-dev/flet#146] [Ideas]
Big flet code very hard support!!!

Example:

In web: html(elements),css(style), js(logic)

in kivy : .kv(elements+style), .py(logic)

BUT in flet   layout+logic must be one one page
<img width="574" alt="Снимок экрана 2022-08-04 в 15 40 34" src="https://user-images.githubusercontent.com/29797481/182849357-a5816201-1556-4b6b-98a7-dd6241944cd9.png">

Flet examples are  needed in rewrite in OOP or create something look like  html+css

**Comment by FeodorFitsner** [comment]
You can better organize the code by move UI pieces into [user controls](https://flet.dev/docs/guides/python/user-controls) and by using [refs](https://flet.dev/blog/control-refs).

If you have a concrete code example I'd love to discuss the ways it could be improved.

**Comment by mrkito** [comment]
@FeodorFitsner 
https://github.com/flet-dev/examples/pull/9

**Reply by ndonkoHenri** [reply]
This will definitely lead to the creation of more files, but nevertheless your approach looks great. 
What do you think @FeodorFitsner ?
